.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I've never used vscode.  Is it typical to check this sort of thing into source control for it?

Reply:
The launch.json settings in this file are very generalized and can apply to any developer who uses VS Code and wants to run the debugger, there isn't anything system specific. It is a matter of opinion on whether it is ignored or added to source control. I am of the opinion it can be added to ease the work of new developers who would be using VS Code, but if you prefer I can remove it from the PR no problem.

Reply:
OK

In the future, please keep things like this in separate PRs but this is fine for now.